recipes/qt/6.x.x/conanfile.py

```diff
@@ -538,11 +539,11 @@ def package_info(self):
         def _get_corrected_reqs(requires):
             reqs = []
             for r in requires:
-                reqs.append(r if "::" in r else "qt%s" % r)
+                reqs.append(r if "::" in r else "Qt6%s" % r)
             return reqs
 
         def _create_module(module, requires=[]):
-            componentname = "qt%s" % module
+            componentname = "Qt6%s" % module
```

Comment:
out of curiosity, why this change ?

Reply:
It makes this piece somewhat more easy (and readable) as now the component names in the conan package align with the names in Qt. Perfectly fine to change it, but to be honest this seemed simpler.

```python
        for m in os.listdir(os.path.join("lib", "cmake")):
            module = os.path.join("lib", "cmake", m, "%sMacros.cmake" % m)
            if os.path.isfile(module):
                self.cpp_info.components[m].build_modules.append(module)
                self.cpp_info.components[m].builddirs.append(os.path.join("lib", "cmake", m))
            else:
                tools.rmdir(os.path.join("lib", "cmake", m))
```